### PR TITLE
feat(v2): Add Turkish translations for theme labels

### PR DESCRIPTION
**Pull request by caglarturali**
## Motivation

To provide default translations in Turkish.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yep.

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

`N/A`

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

`N/A`

### CODE REVIEW

File: packages/docusaurus-theme-classic/codeTranslations/tr.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  "theme.docs.sidebar.expandButtonTitle": "Kenar çubuğunu genişlet",
+  "theme.docs.sidebar.responsiveCloseButtonLabel": "Menüyü kapat",
+  "theme.docs.sidebar.responsiveOpenButtonLabel": "Menüyü aç",
+  "theme.docs.versions.latestVersionLinkLabel": "en son sürüm",
+  "theme.docs.versions.latestVersionSuggestionLabel": "Güncel belgeler için bkz. {latestVersionLink} ({versionLabel}).",
+  "theme.docs.versions.unmaintainedVersionLabel": "Bu, {siteTitle} {versionLabel} dokümantasyonudur ve bakımı sonlanmıştır.",
+  "theme.docs.versions.unreleasedVersionLabel": "Bu, {siteTitle} {versionLabel} sürümü için yayınlanmamış dokümantasyondur.",
+  "theme.lastUpdated.atDate": "{date}",
+  "theme.lastUpdated.byUser": "{user}",
+  "theme.lastUpdated.lastUpdatedAtBy": "En son {atDate} tarihinde {byUser} tarafından güncellendi",
```
Comment (slorber) [inline packages/docusaurus-theme-classic/codeTranslations/tr.json:64]: I think this won't work in all cases.

https://v2.docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-docs

image


We have to handle those 3 cases:
- Last updated by Sébastien
- Last updated on Monday
- Last updated on Monday by Sébastien

Let's imagine user does not want to display the update date, you would end up with a string like `En son tarihinde Sébastien tarafından güncellendi`.

I can't be 100% sure as I'm not speaking Turkish but I feel it's not correct right ? :)

**Review by slorber — CHANGES_REQUESTED** [review]
Thanks for your submission!

Can you please double-check the last updated by labels so that they handle all the possible edge-cases?

File: packages/docusaurus-theme-classic/codeTranslations/tr.json (hunk 1 above)
Comment (caglarturali) [reply packages/docusaurus-theme-classic/codeTranslations/tr.json:64]: Oh, I see. You're right, it won't handle the first two cases correctly. Let me fix it. :)

File: packages/docusaurus-theme-classic/codeTranslations/tr.json [hunk 2]
```diff
@@ -59,9 +59,9 @@
   "theme.docs.versions.latestVersionSuggestionLabel": "Güncel belgeler için bkz. {latestVersionLink} ({versionLabel}).",
   "theme.docs.versions.unmaintainedVersionLabel": "Bu, {siteTitle} {versionLabel} dokümantasyonudur ve bakımı sonlanmıştır.",
   "theme.docs.versions.unreleasedVersionLabel": "Bu, {siteTitle} {versionLabel} sürümü için yayınlanmamış dokümantasyondur.",
-  "theme.lastUpdated.atDate": "{date}",
-  "theme.lastUpdated.byUser": "{user}",
-  "theme.lastUpdated.lastUpdatedAtBy": "En son {atDate} tarihinde {byUser} tarafından güncellendi",
+  "theme.lastUpdated.atDate": "{date} tarihinde",
+  "theme.lastUpdated.byUser": "{user} tarafından",
+  "theme.lastUpdated.lastUpdatedAtBy": "En son{atDate}{byUser} güncellendi",
```
Comment (caglarturali) [inline packages/docusaurus-theme-classic/codeTranslations/tr.json:64]: Looks good to me now. :blush:

Comment (slorber) [reply packages/docusaurus-theme-classic/codeTranslations/tr.json:64]: thanks :)

**Review by slorber — APPROVED** [review]

**Pull request merged**